src/app/services/search.service.ts

```diff
@@ -76,7 +76,25 @@ export class ResearchStudySearchEntry {
   }
   get criteria(): string {
     if (this.resource.enrollment) {
-      return this.resource.enrollment.map((enrollment) => enrollment.display).join(', ');
+      const groupIds = this.resource.enrollment.map((enrollment) => (enrollment.reference as string).substr(1));
+      const characteristics = [];
+      for (const ref of this.resource.contained) {
+        if (ref.resourceType == 'Group' && ref.id in groupIds) {
+          if (ref.characteristic) {
+            //characteristic is an array
+
+            //how criteria is stored in characteristic currently unkown
+            const traits = ref.characteristics.map((trait) => trait.valueCodeableConcept.text).join(', ');
```

Comment:
Unfortunately, I don't think unpacking characteristics is going to be so straight forward. Take this example: https://www.hl7.org/fhir/group-example.json.html where valueCodeableConcept.text would get you "mixed" and "John Smith". In both cases, the value means little without the context of the code. I think building out the complete unpacking will be tedious, and maybe not worth our time right now, but we're going to need to pull out more than just valueCodeableConcept.text. We'll definitely also need at least the exclude boolean to know whether it is inclusion or exclusion criteria.